package.json

```
@@ -118,6 +118,7 @@
   },
   "browser": {
     "lib/index.js": "./lib/jsonld.js",
+    "lib/documentLoaders/node": false,
```

I think its preferred to have an explicit extension:
```suggestion
    "lib/documentLoaders/node.js": false,
```